backend-api/generate_env_file.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+set -eu
+
+if [ $# -ge 1 ] && [ -n "$1" ] ; then
+  SAM_STACK="$1"
+else
+  SAM_STACK="TBC" #default stack name in dev
+fi
+
+echo "Generating .env file for the $SAM_STACK stack"
+
+SELF=$(aws cloudformation describe-stacks --stack-name "$SAM_STACK" --query "Stacks[0].Outputs[?OutputKey=='PublicAPIGWBaseURL'].OutputValue" --output text)
```

Comment:
This I think is a carry over from what we used to call it in teh backend stack.

Please may we use something a bit more explicit?

e.g. REGIONAL_BACKEND_API_URL

Reply:
Going to do pretty dns in follow up pr so this will change then anyway